music_assistant/server/providers/tidal/__init__.py

```diff
@@ -311,19 +311,21 @@ async def get_album_tracks(self, prov_album_id: str) -> list[AlbumTrack]:
         """Get album tracks for given album id."""
         tidal_session = await self._get_tidal_session()
         async with self._throttler:
-            return cast(
-                list[AlbumTrack],
-                [
-                    self._parse_track(
-                        track_obj=track_obj,
-                        extra_init_kwargs={
-                            "disc_number": track_obj.volume_num,
-                            "track_number": track_obj.track_num,
-                        },
-                    )
-                    for track_obj in await get_album_tracks(tidal_session, prov_album_id)
-                ],
-            )
+            with suppress(tidal_exceptions.TooManyRequests):
```

Comment:
this is bad, it supresses the exception and then returns nothing. 
btw: the only call that could potentially give the error is get_album_tracks so imo the backoff/retry logic should be applied on that